main/acle.rst

```diff
@@ -404,6 +408,131 @@ Scalable Vector Extensions (SVE)
 ACLE support for SVE is defined in the Arm C Language Extensions for SVE
 document [SVE-ACLE]_ available on the Arm Developer Website.
 
+.. _ssec-NEONSVEBridge:
+
+NEON-SVE Bridge
+---------------
+
+The NEON-SVE Bridge adds intrinsics that enables setting NEON vectors
+into SVE vectors, and getting NEON vectors from SVE vectors. As the first 128
+bit subvector of an SVE vector is physically a NEON register,
+these intrinsics result in a no-op.
+
+These intrinsics are defined in the header file ``arm_extensions_bridge.h``
+and must be included after ``arm_neon.h`` and ``arm_sve.h``. These
```

Comment:
Good point. How about an approach along the lines of forward declaring the types used by the bridge so that the header stands alone? It may be a small thing but it doesn't seem ideal to `#include` the whole header multiple times into the same compilation unit if it is not necessary.

Reply:
I was thinking more along the lines of
`#define __ARM_NEON_SVE_BRIDGE (__ARM_FEATURE_NEON && __ARM_FEATURE_SVE)``.
``__ARM_FEATURE_NEON` rather than including the header files, but yes the forced order isnt ideal

Reply:
or something like
```
#if !defined(__ARM_FEATURE_NEON) || !defined(__ARM_FEATURE_SVE)
  error
#else
  #define __ARM_NEON_SVE_BRIDGE 1
```

Reply:
Forward-declaring the types sounds like the most palatable solution to me at the moment to make the headers work regardless of order (principle of least surprise for the user), and to avoid double-including them or anything like that.

Reply:
Resolving this conversation as I understand we have decided to force the inclusion of `arm_sve.h` and `arm_neon.h` directly in the header file of the bridge.